src/components/Project/ProjectNew.js

```diff
@@ -0,0 +1,299 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+import {withRouter} from 'react-router';
+import Select from 'react-select';
+
+const styles = {
+  dates: {
+    marginTop: '15px',
+  },
+  participants: {
+    marginBottom: '20px',
+    padding: '0px',
+  },
+  clients: {
+    padding: '0px',
+    margin: '0px 0px 20px 0px',
+  },
+  summary: {
+    height: '400px !important',
+  },
+  control: {
+    control: (base, state) => ({
+      ...base,
+      borderColor: 'lightgrey',
+      boxShadow: `0 0 0 1px lightgrey`,
+      borderRadius: '4px',
+      '&:hover': {
+        borderColor: '#1fbfd7',
+        boxShadow: `1 1 5 10px #1fbfd7`,
+        color: 'white',
+      },
+    }),
+  },
+};
+
+class AddNewProject extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      clientSelect: '',
```

Comment:
I'm going to fix api json files to use camelCase, just for information, can update it afterwards

Reply:
Can we please approve this and merge with develop before we start another thing? This can be a ticket for next week?